crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -37,6 +37,8 @@ const MAX_JOINTS: usize = 256;
 const JOINT_SIZE: usize = std::mem::size_of::<Mat4>();
 pub(crate) const JOINT_BUFFER_SIZE: usize = MAX_JOINTS * JOINT_SIZE;
 
+pub const MESH_VERTEX_OUTPUT: HandleUntyped =
+    HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 8141453348413964287);
```

Comment:
Suspicious integer. Did you just edit some of the numbers to change them a bit? :)

Reply:
... yes.